### PR TITLE
move maps out of the render method

### PR DESCRIPTION
**Pull request by codeBelt**

### CODE REVIEW

**Comment by codeBelt** [comment]
@mthomes
@n8rzz

File: src/views/episodes-page/EpisodesPage.jsx [hunk 1]
```diff
@@ -17,41 +17,49 @@ class EpisodesPage extends React.Component {
...
+        {this._renderTableGroup()}
       </>
     );
   }
+
+  _renderTableGroup() {
+    const { episodeTables } = this.props;
+
+    return episodeTables.map((model) => (
+      <div key={model.title}>
```
Comment (n8rzz) [inline src/views/episodes-page/EpisodesPage.jsx]: what happens if there is another model with a same title?  these `key`s need to be unique

Comment (codeBelt) [reply src/views/episodes-page/EpisodesPage.jsx]: In this case I know that titles will be unique because I create them in the selector.

Comment (n8rzz) [reply src/views/episodes-page/EpisodesPage.jsx]: How would react and/or an implementor know that?  See also:

* https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md
* https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md (this is actually a counter-argument to my index suggestion)

**Pull request merged**